### PR TITLE
Adding AI projects

### PR DESCRIPTION
**Pull request by bjbennet**
SampleAIWorkflows folder includes ingest and chat workflows 
TokenizeDocFunction  folder includes tokenize function needed for both workflows

### CODE REVIEW

**Comment by bjbennet** [comment]
Need to add a readme.md file

File: ai-sample/SampleAIWorkflows/ingest-data/workflow.json [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        "runAfter": {
+          "Convert_string_output_to_JSON": [
+            "SUCCEEDED"
+          ]
+        }
+      },
+      "HTTP": {
+        "type": "Http",
+        "inputs": {
+          "uri": "https://raw.githubusercontent.com/Azure-Samples/azure-search-openai-demo/main/data/Benefit_Options.pdf",
```
[the remark this answers is not in the document]
Comment (bjbennet) [reply ai-sample/SampleAIWorkflows/ingest-data/workflow.json]: updated

File: ai-sample/SampleAIWorkflows/chat-workflow/workflow.json [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+          "system_message_for_search_query": [
+            "SUCCEEDED"
+          ]
+        }
+      },
+      "get_search_query": {
+        "type": "ServiceProvider",
+        "inputs": {
+          "parameters": {
+            "deploymentId": "psrivasModel",
```
Comment (rohithah) [inline ai-sample/SampleAIWorkflows/chat-workflow/workflow.json:112]: parameterize

File: ai-sample/SampleAIWorkflows/ingest-data/workflow.json [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+                    "contentTransfer": {
+                        "transferMode": "Chunked"
+                    }
+                }
+            },
+            "Gets_multiple_embeddings": {
+                "type": "ServiceProvider",
+                "inputs": {
+                    "parameters": {
+                        "deploymentId": "test-embeddings",
```
Comment (rohithah) [inline ai-sample/SampleAIWorkflows/ingest-data/workflow.json:128]: parameterize

File: ai-sample/readme.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+This readme document provides step-by-step instructions on how to enable a **Chat with your Data** Logic Apps project.
+
+## Prerequisites
+
+Before you begin, make sure you have the following prerequisites installed:
+
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Azure Logic Apps extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-logicapps)
+- [Azure Functions extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions)
+- [Azurite extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=Azurite.azurite)
```
Comment (rohithah) [inline ai-sample/readme.md:12]: can you also add instruction on azure search index to create.  we should be specific on the azure search index they need to create (workflows have a dependency on sepcific index column name so we need to have users creating their index with the same column name)

I have added couple of comments on the workflows to parameterize index name etc into app setting - we should add instructions on whre to get those values from open-ai and azure search resources (e.g., openai deploymentId, azure serach index name)

File: ai-sample/SampleAIWorkflows/aisearch_index.json [hunk 5]
```diff
@@ -0,0 +1,123 @@
+{
```
Comment (rohithah) [inline ai-sample/SampleAIWorkflows/aisearch_index.json:1]: I would keep this out side of the workflow project - maybe under new folder (e.g., deployment)

**Review by rohithah — APPROVED** [review]

**Pull request merged**